amd_openvx_extensions/CMakeLists.txt

```diff
@@ -119,8 +119,36 @@ else()
 endif()
 
 if(AMDRPP_FOUND AND GPU_SUPPORT)
-    add_subdirectory(amd_rpp)
-    message("-- ${Green}AMD Radeon Performance Primitives -- amd_rpp module added with GPU Support${ColourReset}")
+    #find the RPP backend type
```

Comment:
@asalmanp this change is still letting OpenCL to build with HIP RPP

```
* VX RPP Library
````
	linux-vdso.so.1 (0x00007ffe33be8000)
	libamd_rpp.so => /opt/rocm/rpp/lib/libamd_rpp.so (0x00007f5c31a20000)
	libboost_filesystem.so.1.71.0 => /lib/x86_64-linux-gnu/libboost_filesystem.so.1.71.0 (0x00007f5c319e7000)
	libOpenCL.so.1 => /opt/rocm/opencl/lib/libOpenCL.so.1 (0x00007f5c317df000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f5c315fd000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f5c315e2000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f5c313f0000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f5c313cb000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f5c3127c000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f5c31271000)
	libamdhip64.so.4 => /opt/rocm-4.3.1/hip/lib/libamdhip64.so.4 (0x00007f5c304c1000)
	libomp.so => /opt/rocm-4.3.1/llvm/bin/../lib/libomp.so (0x00007f5c301cc000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f5c301c4000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f5c32264000)
	libhsa-runtime64.so.1 => /opt/rocm-4.3.1/hip/lib/../../lib/libhsa-runtime64.so.1 (0x00007f5c2fd44000)
	libnuma.so.1 => /lib/x86_64-linux-gnu/libnuma.so.1 (0x00007f5c2fd37000)
	libhsakmt.so.1 => /opt/rocm-4.3.1/hip/lib/../../lib/libhsakmt.so.1 (0x00007f5c2fb0c000)
	libelf.so.1 => /lib/x86_64-linux-gnu/libelf.so.1 (0x00007f5c2faf0000)
	libz.so.1 => /usr/local/lib/libz.so.1 (0x00007f5c2face000)

````
```

Reply:
I have set the CI to run again, I will wait for the results on Ubuntu 20 for setup built with HIP backend and MIVisionX built with OpenCL backend.